### PR TITLE
Create com.github.muriloventuroso.pdftricks.json

### PR DESCRIPTION
**Pull request by muriloventuroso**

### CODE REVIEW

File: com.github.muriloventuroso.pdftricks.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        "--socket=x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro"
+    ],
+    "build-options": {
```
[the remark this answers is not in the document]
Comment (barthalion) [reply com.github.muriloventuroso.pdftricks.json]: To expand, they are set by default.

**Comment by barthalion** [comment]
bot, build com.github.muriloventuroso.pdftrick
testbot, build com.github.muriloventuroso.pdftrick

File: com.github.muriloventuroso.pdftricks.json [hunk 2]
```diff
@@ -11,37 +11,33 @@
         "--socket=x11",
         "--device=dri",
         "--socket=wayland",
-        "--filesystem=home",
         "--talk-name=ca.desrt.dconf",
-        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
```
Comment (bilelmoussaoui) [inline com.github.muriloventuroso.pdftricks.json]: Those are needed if you gsettings. If not, you need to remove the rest of dconf permissions too.

**Comment by barthalion** [comment]
bot, build com.github.muriloventuroso.pdftrick
testbot, build com.github.muriloventuroso.pdftrick

**Comment by barthalion** [comment]
bot, build com.github.muriloventuroso.pdftricks
testbot, build com.github.muriloventuroso.pdftricks

**Comment by barthalion** [comment]
Tried compressing a random PDF:

```

**Comment by barthalion** [comment]
bot, build com.github.muriloventuroso.pdftricks
testbot, build com.github.muriloventuroso.pdftricks

**Comment by muriloventuroso** [comment]
I'm working on the progress bar, so I finish releasing an update also for flatpak.
I will update the manifest to clear unnecessary commands.
That icon that is missing is "process-completed", but in the flatpak package that I generated on my computer the icon appears normally.

**Comment by bilelmoussaoui** [comment]
Because you're using elementary icon theme where the icon exists. I would suggest to patch the application here to force using the elementary gtk/icon theme to avoid having an application that doesn't look like what you show on the screenshots in the appdata file.

**Comment by barthalion** [comment]
testbot, build com.github.muriloventuroso.pdftricks

**Comment by barthalion** [comment]
LGTM. I'll wait with the merge as import of KDE repo affects stability of builds.

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/com.github.muriloventuroso.pdftricks

Thanks!

**Pull request closed without merging**